setup.py

```diff
@@ -23,10 +23,11 @@ def local_scheme(version):
     #long_description="""TODO""",
     license="BSD",
     setup_requires=["setuptools_scm"],
-    install_requires=["nmigen"],
+    install_requires=["nmigen~=0.1.rc1"],
```

Comment:
I don't think this should be part of this PR. Is there a special feature from nmigen 0.1rc1 that this PR requires?
In any case I don't think it makes sense to track releases yet, since everything is still in flux.

Reply:
So far the only issue with 0.1RC1 I have encountered is that what was called `nmigen.tools` is now `nmigen.utils`, and RC1 doesn't seem to give any backward compatibility. Since Minerva has already been updated to RC1, for the purpose of HeavyX nmigen-stdio does require 0.1RC1 in order to bulid the bitstream. 

I also noticed nmigen-soc is using 0.1RC1. @whitequark may I know why 0.1RC1 is used there?

Reply:
Probably an oversight. Both nmigen-stdio and nmigen-soc should require `nmigen~=X` where X is the latest stable version, currently 0.1.